plugins/fe_events_direct/db/services.php

```diff
@@ -38,4 +41,11 @@
         'loginrequired' => true,
         'ajax' => true
     ),
+    'local_fe_events_direct_user_id_handler' => array(
```

Comment:
I didn't really notice this in the previous PR for `local_fe_events_direct_content_loaded_handler`, but the exposed function names are typically more like `{plugin type}_{plugin name}_{something verbish}`. For example, in our current `moodle-local_raisecli` plugin we have names such as `local_raisecli_enable_self_enrolment_method` and `local_raisecli_get_role_by_shortname`.